base/strings/basic.jl

```diff
@@ -280,20 +280,19 @@ isascii(s::AbstractString) = all(isascii, s)
 promote_rule{S<:AbstractString,T<:AbstractString}(::Type{S}, ::Type{T}) = String
 
 """
-    isxdigit(c::Union{Char,AbstractString}) -> Bool
+    isxdigit(c::Char) -> Bool
 
-Tests whether a character is a valid hexadecimal digit, or whether this is true for all elements of a string.
+Tests whether a character is a valid hexadecimal digit.
 
 ```jldoctest
-julia> isxdigit("abc")
+julia> isxdigit('a')
 true
 
-julia> isxdigit("0x9")
+julia> isxdigit('z')
```

Comment:
The `0` and `x` cases where interesting to show that the prefix wasn't considered as a valid hexadecimal digit. Say this in the docstring?